src/MonitoringFunctions/Functions/DownloadPs1.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+
+namespace MonitoringFunctions.Functions
+{
+    public static class DownloadPs1
```

Comment:
very minor: Type names should not be "commands" or "actions".  Instead, please consider ```Ps1Downloader```